packages/meso-js/README.md

```diff
@@ -408,13 +417,66 @@ const { destroy } = transfer({ ... });
 destroy(); // The meso iframe is unmounted. No more events/callbacks will fire.
 ```
 
-### Headless Signature
+### Authentication Strategies
+
+Depending on your integration, you may have different requirements for users
+authenticating with Meso. In all scenarios, the user will still be required to
+perform two-factor authentication (2FA) and, in some cases provide
+email/password.
+
+#### Wallet verification
+
+By default, users are prompted to sign a message with
+their wallet to prove ownership.
+
+<details>
+  <summary><strong>Example</strong></summary>
+
+```ts
+{
+  // ...
+  authenticationStrategy: AuthenticationStrategy.WALLET_VERIFICATION,
+}
+```
+
+</details>
+
+#### Headless wallet verification
+
+In cases such as embedded wallets, message signing may need be transparent to
+the user. In these cases, passing will allow you to perform
+message signing yourself in the background.
+
+<details>
+  <summary><strong>Example</strong></summary>
+
+```ts
+{
+  // ...
+  authenticationStrategy: AuthenticationStrategy.HEADLESS_WALLET_VERIFICATION,
+}
+```
+
+</details>
+
+#### Bypass wallet verification
+
+In the case where pre-deployment smart contract wallets are being used and
+wallet verification cannot be performed, you can skip wallet verification
+altogether and rely on Meso's two-factor authentication and email/password
+mechanisms.
```

Comment:
Nit: may not always be email/password. Maybe `rely on Meso's two-factor authentication and other authentication mechanism`?

Reply:
I like this. Changing